### PR TITLE
docs: add guide for building a simple langchain

### PR DESCRIPTION
**Pull request by swarnimarun**

### CODE REVIEW

File: docs/guides/langchain.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Building a langchain AI bot for conversation using Prem-Operator
+
+## Requirements
+
+- Kubernetes deployment with atleast 1 GPU based node, if you are using CPU based node consider using smaller models
+- Prem Operator v1.0.0 or above
```
Comment (richiejp) [inline docs/guides/langchain.md:6]: I don't think we should start with v1.0.0, people's expectations start to get wild.

Comment (swarnimarun) [reply docs/guides/langchain.md:6]: Good point, I was also think to make it 0.1 release or something?

Comment (swarnimarun) [inline docs/guides/langchain.md:6]:
```suggestion
- Prem Operator v0.1.0 or above
```

Comment (richiejp) [reply docs/guides/langchain.md]: I'd just stick with 0.10.1

**Pull request merged**